### PR TITLE
Deleted unecessaries imports optimize logic in if and for

### PR DESCRIPTION
**Pull request by Barbacraft**
## Issue

[Code Quality] improve the code quality of Nacos 2.0. #5726

## What is the purpose of the change

improve the code quality of Nacos 2.0.

## Brief changelog

Remove unnecessary imports and improvre logic

### CODE REVIEW

**Comment by KomachiSion** [comment]
Please sign CLA first.

File: client/src/main/java/com/alibaba/nacos/client/config/common/GroupKey.java [hunk 1]
```diff
@@ -16,7 +16,8 @@
 
 package com.alibaba.nacos.client.config.common;
 
-import com.alibaba.nacos.common.utils.StringUtils;
+import com.alibaba.nacos.common.main.java.utils.StringUtils;
```
Comment (KomachiSion) [inline client/src/main/java/com/alibaba/nacos/client/config/common/GroupKey.java:19]: Why use this?

**Review by KomachiSion — CHANGES_REQUESTED** [review]

**Review by Deweywilliams — APPROVED** [review]
Unauthorized user need real name and address to log in this phone without any links involved

**Comment by KomachiSion** [comment]
CLA not sign for a long time.

**Pull request closed without merging**